### ISSUE TITLE
#1616749757------------- Buttons must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#410]
<strong>Source:</strong>`<button aria-label="" aria-live="polite" class="button-padding--shrunk icon-end-empty" type="button"><calcite-icon class="icon icon--start" aria-hidden="true" icon="square" scale="s"></calcite-icon></button>`
<strong>Element Location: </strong>["#running-button-4284171ab7c845dfbbbe8dfdf5b243ec > calcite-button[icon-start=\"square\"][kind=\"neutral\"][appearance=\"transparent\"]","button"]
<strong>Impact: </strong>critical
<strong>failureSummary:</strong> Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"Tags: [Ljava.lang.String;@5f2bd6d9
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
